CHANGELOG.next.asciidoc

```diff
@@ -75,13 +72,6 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 
 *Filebeat*
 
-- Add `text/csv` decoder to `httpjson` input {pull}28564[28564]
```

Comment:
These all should have gone into 8.1 or earlier. It's too difficult to go back and figure out which release they were actually in. No idea why they were left behind here.